src/deployment/flavors.md

```diff
@@ -3,41 +3,226 @@ title: Creating flavors for Flutter
+default configurations that are already available and the new configurations 
+for the `free` scheme. 
+* Under the **Info** tab at the end of the 
+**Configurations** dropdown list, click the plus button and duplicate 
+each configuration name (Debug, Release, and Profile). 
+Duplicate the listed production configurations, once for each environment. 
+
+![Step 3 Xcode image](/assets/images/docs/flavors/step3-ios-build-config.png){:width="100%"}
+{{site.alert.note}}
+  Your configurations should be based on your **Debug.xconfig** or **Release.xcconfig**
+  file, not the **Pods-Runner.xcconfigs**. You can check this by expanding the configuration names. 
+{{site.alert.end}}
+
+</li>
+<li markdown="1">
+
+To match the development flavor, add `-free` 
+at the end of each new configuration name. 
+
+</li>
+<li markdown="1">
+
+Change the `free` scheme to match the development 
+build configurations already created.
+* In the **Runner** project, click **Manage Schemes…** and a pop up window opens. 
+* Double click the development scheme. In the next step 
+(as shown in the screenshot), you’ll modify each scheme 
+to match its dev build configuration:
+
+![Step 5 Xcode image](/assets/images/docs/flavors/step5-ios-scheme.png){:width="100%"}
+
+</li>
+</ol>
+
+## Using flavors in iOS
+
+Now that you’ve set up your development flavor, 
+you can, for example, add different product bundle identifiers per flavor. 
+A _bundle identifier_ uniquely identifies your application. 
+In this example, we set the **Debug-free-example** value to equal 
+`com.flavor-test.dev`. 
+
+<ol markdown="1">
+<li markdown="1">
+
+Change the app bundle identifier to differentiate between schemes. 
+In **Product Bundle Identifier**, append `.free` to each -free scheme value.
+
+![Step 1 using flavors image.](/assets/images/docs/flavors/step1-using-flavors.png){:width="100%"}  
+
+</li>
+<li markdown=1>
+
+In the **Build Settings**, set the **Product Name** value to match each flavor. 
+
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}  
+
+</li>
+<li markdown=1>
+
+Add the display name to **info.plist**. Create a new key in **info.plist** called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
+
+![Step 3 using flavors image.](/assets/images/docs/flavors/step3-using-flavors.png){:width="100%"}    
+
+</li>
+</ol>
+
+Now you have set up your development flavor by making a `free` scheme 
+in Xcode and setting the build configurations for that scheme. 
+
+For more information, skip to the [“Launching your app flavors”][] 
+section at the end of this document.
+
+### Plugin configurations
+
+If your app has a Flutter plugin, you will need to update the `ios/Podfile`. 
+
+1. In `ios/Podfile` change the default of **Debug**, **Profile**, and **Release** 
+to match the Xcode build configurations for the `free` scheme.
+
+```ruby
+project 'Runner', {
+  'Debug-free' => :debug,
+  'Debug-production' => :debug,
```

Comment:
Are you making a `production` flavor too?  I don't think that's now mentioned anywhere.